akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -2285,6 +2285,281 @@ private[akka] final class StatefulMap[S, In, Out](
   override def toString = "StatefulMap"
 }
 
+/**
+ * INTERNAL API
+ */
+private[akka] final case class StatefulMapAsync[S, In, Out](parallelism: Int)(
```

Comment:
Its ordered, or maybe a `StatefulMapAsyncUnordered` then?

Reply:
sure, after confirm current design is acceptable, i will make unordered version.

Reply:
fs2 and zio both are parallelism 1 here, maybe that's should be a good start